subgraphs/spotify/src/index.ts

```diff
@@ -46,98 +42,85 @@ const loggerMiddleware = morgan(
   }
 );
 
-async function main() {
-  let typeDefs = gql(
-    readFileSync('schema.graphql', {
-      encoding: 'utf-8',
-    })
-  );
-  const schema = buildSubgraphSchema({
-    typeDefs,
-    resolvers,
-  });
-  const app = express();
-  const httpServer = http.createServer(app);
+const app = express();
+Sentry.init({
+  dsn: process.env.SENTRY_URL,
+  integrations: [
+    // enable HTTP calls tracing
+    new Sentry.Integrations.Http({
+      tracing: true,
+    }),
+    // enable Express.js middleware tracing
+    new Sentry.Integrations.Express({
+      app,
+    }),
+  ],
+});
+
+export const contextFunction = async ({ req }) => {
   const defaultCountryCode = readEnv('DEFAULT_COUNTRY_CODE', {
     defaultValue: 'US',
   });
 
-  const server = new ApolloServer<ContextValue>({
-    schema,
-    introspection: true,
-    logger,
-    plugins: [
-      {
-        async requestDidStart() {
-          return {
-            responseForOperation: async (operation) => {
-              const { request, contextValue } = operation;
-              if (!contextValue.mock) return;
-
-              const { query, variables, operationName } = request;
-              const response = await execute({
-                schema: addMocksToSchema({
-                  schema: buildSubgraphSchema({ typeDefs }),
-                  mocks,
-                  preserveResolvers: true,
-                }),
-                document: parse(query),
-                contextValue,
-                variableValues: variables,
-                operationName,
-              });
-
-              return {
-                http: request.http,
-                body: { kind: 'single', singleResult: response },
-              } as GraphQLResponse;
-            },
-          };
-        },
+  checkRouterSecret(req.headers['router-authorization'] as string);
+  const token = req.get('authorization');
+
+  if (!token) {
+    const userIdForMocks = req.get('x-graphos-id') ?? 'default';
```

Comment:
```suggestion
    const userIdForMocks = req.get('x-graphos-id') ?? 'shared';
```

What about calling the default fallback id `shared` since the state for all users without an id will be "shared". Thoughts?